index.php

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <?php $base = "../../" ?>
+    <base href="../../">
+    <script src="js/jquery-2.2.4.min.js"></script>
```

Comment:
why do we need jquery?

Reply:
It's required for the "Play Now!" button on the bottom of the page.